server.js

```diff
@@ -1,11 +1,137 @@
 'use strict';
 
-const { ApolloServer } = require('apollo-server');
+/* eslint-disable no-param-reassign */
```

Comment:
we need this to stop eslint complaining about lines 47-49

Reply:
can we wrap this around just the offending lines instead of the entire file?